control/mon_control.txt

```diff
@@ -156,37 +156,158 @@ Yellow Plant 0 0 0
 1967 -1 0 0
 
 ##### MVPs and Dangerous Monsters #####
+##MOB NAME : MVP
 Amon Ra 0 1 0
+Amdarias 0 1 0
```

Comment:
Amdarais not Amdarias

Reply:
RMS is Amdarias hahhaah  
ttp://ratemyserver.net/index.php?mvp=1&page=mob_db&mob_search=Search&page_num=1

Reply:
Amdarias (Re-Stats)   (MG_AMDARAIS)   Mob-ID#2476
Amdarais   (MG_AMDARAIS)   Mob-ID#2476
= = I'll add them both.